server/athenian/api/controllers/account.py

```diff
@@ -37,46 +37,6 @@
 jira_url_template = os.getenv("ATHENIAN_JIRA_INSTALLATION_URL_TEMPLATE")
 
 
-@cached(
```

Comment:
moved some stuff so that graphql code doesn't need to access `controllers` package